src/main/java/net/irisshaders/iris/shaderpack/parsing/BooleanParser.java

```diff
@@ -0,0 +1,150 @@
+package net.irisshaders.iris.shaderpack.parsing;
+
+import net.irisshaders.iris.Iris;
+import net.irisshaders.iris.shaderpack.option.values.OptionValues;
+
+import java.util.EmptyStackException;
+import java.util.Stack;
+
+public class BooleanParser {
+	private enum OPERATION {
```

Comment:
enums in this codebase are typically in UpperCamelCase (though their elements are SCREAMING_SNAKE_CASE)

Reply:
will change that